pydis_site/apps/api/serializers.py

```diff
@@ -41,6 +42,33 @@ class Meta:
         fields = ('name', 'data')
 
 
+class ListBumpedThreadSerializer(ListSerializer):
+    """Custom ListSerializer to override to_representation() when list views are triggered."""
+
+    def to_representation(self, objects: list[BumpedThread]) -> int:
+        """
+        Used by the `ListModelMixin` to return just the list of bumped thread ids.
+
+        We want to only return the thread_id attribute, hence it is unnecessary
+        to create a nested dictionary.
```

Comment:
Please avoid using "we", "us", "I", etc. in documentation.
```suggestion
        Only the thread_id field is useful, hence it is unnecessary to create a nested dictionary.
```